### PR TITLE
New KB article: Custom JpegImageConverter

### PR DESCRIPTION
**Pull request by velikovmartin**

### CODE REVIEW

File: knowledge-base/create-custom-jpeg-image-converter-net-standard.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+<thead>
+	<tr>
+		<th>Product Version</th>
+		<th>Product</th>
+		<th>Author</th>
+	</tr>
+</thead>
+<tbody>
+	<tr>
+		<td>2020.1.506</td>
```
Comment (DKaramfilov) [inline knowledge-base/create-custom-jpeg-image-converter-net-standard.md:22]: The version looks weird. Is this the correct version?

File: knowledge-base/create-custom-jpeg-image-converter-net-standard.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## Solution
+
+The following code snippets demonstrates how to create a custom implementation of the JpegImageConverterBase abstract class using the [SixLabors.ImageSharp](https://github.com/SixLabors/ImageSharp) library and set it to the JpegImageConverter property of the FixedExtensibilityManager. We are using the ImageSharp library to convert the images from one of the library`s supported formats to Jpeg and to change their quality if it is set.
+
+#### __C#__ Create a custom implementation inheriting the JpegImageConverterBase abstract class
+
+{{region kb-create-custom-jpeg-image-converter1}}
+
+	internal class CustomJpegImageConverter : JpegImageConverterBase
```
Comment (DKaramfilov) [inline knowledge-base/create-custom-jpeg-image-converter-net-standard.md:41]: the code is not formated correctly (too many tabs)

File: knowledge-base/create-custom-jpeg-image-converter-net-standard.md (hunk 1 above)
Comment (velikovmartin) [reply knowledge-base/create-custom-jpeg-image-converter-net-standard.md:22]: I copy-pasted it from the DLL's properties. It is the DLL from the 6th of May. Should it be from a specific LIB or Release?

File: knowledge-base/create-custom-jpeg-image-converter-net-standard.md (hunk 2 above)
Comment (velikovmartin) [reply knowledge-base/create-custom-jpeg-image-converter-net-standard.md:41]: Тhis does not affect the appearance of the documentation but it improves the appearance in Markdown Monster.

**Pull request merged**